### DISCUSSION TITLE
Implementation of clustering in CoxPH

### DISCUSSION THREAD

**Discussion by jchts01** [CamDavidsonPilon/lifelines#1307] [Q&A]
I used the clustering function with CoxPH fitter while working on a set of medical data (on readmission) to account for the same patients having more than one episode, but I am uncertain as to how clustering was handled in lifelines. My question is: does applying 'cluster_col' invoke only the use of sandwich variance estimates? Or does it employ a shared frailty model as well? Thanks.

**Comment by CamDavidsonPilon** [comment]
Hi @jchts01 - it only invokes sandwich variance estimates

**Reply by jchts01** [reply]
Understood, thanks!
